testing/test_doctest.py

```diff
@@ -146,9 +146,9 @@ def test_encoding_ascii(self, testdir):
     def test_encoding_latin1(self, testdir):
         """Test support for --doctest-encoding option.
         """
-        testdir._makefile(".txt", ["""
-            >>> 'üäö'
-            'üäö'
+        testdir._makefile(".txt", [u"""
+            >>> len(u'üäö')
+            3
```

Comment:
i propose making those a format string adding u in front of it for py2, and leaving it out for py3
that way the output can be tested